packages/apps/app-dashboard/src/hooks/user-dashboard/useAuthInfo.ts

```diff
@@ -123,6 +130,7 @@ export const useSetAuthInfo = (): UseSetAuthInfo => {
     } catch (err) {
       const error = err as Error;
       console.error('Error storing auth info:', error);
+      Sentry.captureException(error);
       setError(error);
 
       throw error;
```

Comment:
nit: Because error is re-thrown, we are likely capturing or handling it afterwards, therefore making this capture a duplicated or a non-relevant one

Still, better to have it and be a false alarm than not having it